### DISCUSSION TITLE
how to import skin on osu Lazer linux chromebook

### DISCUSSION THREAD

**Discussion by cool271** [ppy/osu#28722] [Q&A]
please help

**Comment by sineplusx** [comment]
> please help

If you want to get help you should be a bit more descriptive about the issue you have.
Does it happen with all skins or just some of them? How have you tried to import skins so far?

You can import skins by dragging and dropping a skin file into the osu! window (if you are in windowed mode), or by using the "Import files" dialog in the "Debug" section of the settings.

**Reply by cool271** [reply]
I can't drag or drop any skin on osu Lazer, also I'm on chromebook so I use Linux to download it

**Reply by Stoppedpuma** [reply]
As @sineplusx mentioned, please try using the import function, https://github.com/ppy/osu/discussions/28742#discussioncomment-9961388 as shown on your other discussion you made.
